setup.cfg

```diff
@@ -79,7 +79,7 @@ all =
     ipython
     pytest
 docs =
-    sphinx
+    sphinx<4
```

Comment:
I made #11763 to unpin in dev, but I don't think we need to do it here as the stable branch will get overwritten when 4.3 is out anyway.